README.md

```diff
@@ -79,3 +79,29 @@ yarn test:e2e
 ```bash
 yarn test
 ```
+
+## Open Dollar App - Docker Deployment Guide
+
+This guide walks you through the steps to run the Open Dollar App using Docker Compose based on the provided docker-compose.yml and Dockerfile.
+##### Pre-requisites:
+Ensure Docker and Docker Compose are installed on your system.
+##### Dockerfile Overview:
```

Comment:
The instructions here do not clearly state how to run the container locally.

Reply:
At this point, explaining the container is unnecessary. Only the simple instructions for usage should be included

Reply:
@pi0neerpat Thanks for the comment and I think it makes sense and now I only give 3 topics:

1. Docker file overview
2. A command to run the container and create the image locally
3. A command to stop the application in Docker Compose